EBA/Taschenrechner/Informatikrechner/Zahlensystem.cs

```diff
@@ -0,0 +1,192 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Taschenrechner
+{
+    public partial class fm_Zahlensystem : BaseForm
+    {
+        string eingabe = string.Empty;
+        public fm_Zahlensystem()
+        {
+            InitializeComponent();
+        }
+
+        public string Eingabe()
```

Comment:
I have read the class before. this code exists twice I think.

Reply:
see above